apps/nextjs/src/app/aila/page-contents.tsx

```diff
@@ -8,10 +8,14 @@ import Layout from "@/components/AppComponents/Layout";
 import { ChatProvider } from "@/components/ContextProviders/ChatProvider";
 import { useReactScan } from "@/hooks/useReactScan";
 import LessonPlanTrackingProvider from "@/lib/analytics/lessonPlanTrackingContext";
+import { useChatStore } from "@/stores/chatStore";
+import useStoreReset from "@/stores/chatStore/hooks/useStoreReset";
 
 const ChatPageContents = ({ id }: { readonly id: string }) => {
   useReactScan({ component: LessonPlanDisplay, interval: 10000 });
-
+  const setId = useChatStore((state) => state.setId);
+  setId(id);
```

Comment:
Will this set the ID every thing the component renders? That sounds like the kind of pattern we're trying to get away from